### PR TITLE
css-brochure-form - Junou Lee

### PR DESCRIPTION
**Pull request by junoulee**
image

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+h3 {
+  text-align: center;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:5]: This is not necessary

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  margin-left: 74%;
+  padding: 8px 25px;
+  border-radius: 30px;
+  background-color: rgb(255 51 0 / 78.3%);
+  border-color: rgb(255 123 0 / 78.3%);
+  border-style: solid;
+  color: white;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:41]: This is not necessary

File: css-brochure-form/styles.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+input#email::placeholder,
+input#name::placeholder,
+input#hiring::placeholder {
+  padding-left: 1.5%;
+  color: rgb(169 167 167);
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:62]: This is not necessary

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-brochure-form/styles.css (hunk 1 above)
Comment (junoulee) [reply css-brochure-form/styles.css:4]: `body {
  font-family: "Brandon Grotesque", sans-serif;
  font-style: normal;
  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
}

h3 {
  text-align: center;
  font-weight: 500;
  font-size: 1.2rem;
  color: rgb(255 51 0 / 78.3%);
}

form {
  width: 50%;
  margin: auto;
}

input#email,
input#name,
input#hiring,
select,
option {
  margin: auto;
  width: 95%;
  padding-top: 9px;
  padding-bottom: 9px;
  border-color: rgb(213 212 212);
  border-radius: 6px;
  border-style: solid;
  border-width: 1.8px;
}

button {
  margin-left: 74%;
  padding: 8px 25px;
  border-radius: 30px;
  background-color: rgb(255 51 0 / 78.3%);
  border-color: rgb(255 123 0 / 78.3%);
  border-style: solid;
  color: white;
  font-weight: 500;
  font-size: 0.9rem;
}

label {
  font-weight: 525;
  font-size: 0.8rem;
  margin-left: 2%;
}

input#email::placeholder,
input#name::placeholder,
input#hiring::placeholder {
  padding-left: 1.5%;
  color: rgb(169 167 167);
  font-style: normal;
  font-weight: normal;
  font-size: 1rem;
}

select {
  background-color: rgb(245 243 243);
  border-color: rgb(202 200 200 / 77.5%);
}
`

Comment (codingcodymiller) [reply css-brochure-form/styles.css:4]: What is this comment trying to convey? Can you explain what you are trying to say with words?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-brochure-form/styles.css (hunk 1 above)
Comment (junoulee) [reply css-brochure-form/styles.css:4]: oh that I fixed it, I just pasted the updated code

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**